vtex/faststore
# Feature/region agora vai

## What's the purpose of this pull request?
Adds regionalization.

This PR represents the code that has already been merged on `next`.

Tests incoming.

## How it works? 
<em>Tell us the role of the new feature, or component, in its context.</em>

## How to test it?
https://sfj-8d671be--carrefourbrfood.vtex.app/

Set the CEP to "01309-010"

Search for "ovo de páscoa"

If product availability hasn't changed much, it should look like this:
![Screen Shot 2021-04-09 at 17 15 10](https://user-images.githubusercontent.com/5691711/114235896-36787300-9957-11eb-8c44-9e76e5703fd0.png)


Change the CEP to "22250-040".

The products should look like this:
![Screen Shot 2021-04-09 at 17 14 15](https://user-images.githubusercontent.com/5691711/114235763-0630d480-9957-11eb-9bb9-6669261ff629.png)

Click on the first one. The product should be available:
![Screen Shot 2021-04-09 at 17 16 13](https://user-images.githubusercontent.com/5691711/114235960-4d1eca00-9957-11eb-83c9-eca1eb0bd5d6.png)

Change the CEP back to "01309-010". The product should be unavailable:
![Screen Shot 2021-04-09 at 17 16 37](https://user-images.githubusercontent.com/5691711/114236023-6162c700-9957-11eb-94a2-da61504e310f.png)


## References
<em>Spread the knowledge: is there any content you used to create this PR that is worth sharing?</em>  

<em>Extra tip: adding references to related issues or mentioning people important to this PR may be good for the documentation and reviewing process</em>
